src/TaxToken.sol

```diff
@@ -66,26 +69,26 @@ contract TaxToken {
         uint256 maxWalletSizeInput,
         uint256 maxTxAmountInput
     ) {
-        _paused = false;    // ERC20 Pausable global state variable, initial state is not paused ("unpaused").
+        _paused = false;
         _name = nameInput;
         _symbol = symbolInput;
         _decimals = decimalsInput;
         _totalSupply = totalSupplyInput * 10**_decimals;
 
-        // Create a uniswap pair for this new token
+        // Create a uniswap pair for this new token.
         address UNISWAP_V2_PAIR = IUniswapV2Factory(
             IUniswapV2Router01(UNIV2_ROUTER).factory()
         ).createPair(address(this), IUniswapV2Router01(UNIV2_ROUTER).WETH());
  
         senderTaxType[UNISWAP_V2_PAIR] = 1;
         receiverTaxType[UNISWAP_V2_PAIR] = 2;
 
-        owner = msg.sender;                                         // The "owner" is the "admin" of this contract.
-        balances[msg.sender] = totalSupplyInput * 10**_decimals;    // Initial liquidity, allocated entirely to "owner".
+        owner = msg.sender;                                         
+        balances[msg.sender] = totalSupplyInput * 10**_decimals;
         maxWalletSize = maxWalletSizeInput * 10**_decimals;
         maxTxAmount = maxTxAmountInput * 10**_decimals;
 
-        // TODO: Add to main-net deployment.
+        // TODO: Add before main-net deployment.

```

Comment:
Is it before? Doesnt the contract have to be deployed to call those functions? Dont want this comment to be misleading.

Reply:
Could go either way, I just thought it made more sense to add that line before we do a main-net deployment